myFormat.js

```diff
@@ -0,0 +1,141 @@
+const board = [
+    ['T', 'T', '.', 'F'],
+    ['T', '.', '.', '.'],
+    ['.', '.', '.', '.'],
+    ['R', '.', '.', 'W']
+  ];
+  
+//------------------------------
+
+const robot = {
+    x: 0,
+    y: 0,
+    dir: 'up'
+  };
+ 
+//------------------------------
+
+  let moves = 0;
+  let turns = 0;
+  let applesEaten = 0;
+  
+
+//------------------------------
+
+const trailIndicators = {
+    left: '←',
+    right: '→',
+    up: '↑',
+    down: '↓'
+  };
+
+//------------------------------
+
+board.reverse();
+
+//------------------------------
+
+  function move() {
+
+    let x = robot.x;
+    let y = robot.y;
+
+    if (robot.dir === 'up') {
+      robot.dir = 'up';
+      if (y < board.length - 1) {
+        y = y + 1
+      } else {
+        y = y
```

Comment:
You can leave the `else` part altogether here because `y = y;` doesn't change anything. In the ternary operator version it was necessary because you are required to have an expression after the `:` symbol.

BTW, don't forget the semi-colons `;`